code/game/objects/items/stacks/rods.dm

```diff
@@ -73,3 +73,12 @@ var/global/list/datum/stack_recipe/rod_recipes = list ( \
 
 /obj/item/stack/rods/cyborg/update_icon()
 	return
+
+/obj/item/stack/rods/ten
+	amount = 10
+
+/obj/item/stack/rods/twentythree

```

Comment:
I don't care about the hat balance one, that's completely fine, it's this stack one I object to the most.